docs/src/extend/custom-rules.md

```diff
@@ -242,39 +270,38 @@ ruleTester.run("my-rule", rule, {
 
 ```js
 context.report({
     node: node,
     message: "Missing semicolon",
-    fix: function(fixer) {
+    fix(fixer) {
         return fixer.insertTextAfter(node, ";");
     }
 });
 ```
 
-Here, the `fix()` function is used to insert a semicolon after the node. Note that a fix is not immediately applied, and may not be applied at all if there are conflicts with other fixes. After applying fixes, ESLint will run all of the enabled rules again on the fixed code, potentially applying more fixes. This process will repeat up to 10 times, or until no more fixable problems are found. Afterwards, any remaining problems will be reported as usual.
+Here, the `fix()` function is used to insert a semicolon after the node. Note that a fix is not immediately applied, and may not be applied at all if there are conflicts with other fixes. After applying fixes, ESLint will run all the enabled rules again on the fixed code, potentially applying more fixes. This process will repeat up to 10 times, or until no more fixable problems are found. Afterward, any remaining problems will be reported as usual.
 
-**Important:** The `meta.fixable` property is mandatory for fixable rules. ESLint will throw an error if a rule that implements `fix` functions does not [export](#rule-basics) the `meta.fixable` property.
+**Important:** The `meta.fixable` property is mandatory for fixable rules. ESLint will throw an error if a rule that implements `fix` functions does not [export](#rule-structure) the `meta.fixable` property.
 
 The `fixer` object has the following methods:
 
-* `insertTextAfter(nodeOrToken, text)` - inserts text after the given node or token
-* `insertTextAfterRange(range, text)` - inserts text after the given range
-* `insertTextBefore(nodeOrToken, text)` - inserts text before the given node or token
-* `insertTextBeforeRange(range, text)` - inserts text before the given range
-* `remove(nodeOrToken)` - removes the given node or token
-* `removeRange(range)` - removes text in the given range
-* `replaceText(nodeOrToken, text)` - replaces the text in the given node or token
-* `replaceTextRange(range, text)` - replaces the text in the given range
+* `insertTextAfter(nodeOrToken, text)`: Insert text after the given node or token.
+* `insertTextAfterRange(range, text)`: Insert text after the given range
+* `insertTextBefore(nodeOrToken, text)`: Insert text before the given node or token
+* `insertTextBeforeRange(range, text)`: Insert text before the given range
+* `remove(nodeOrToken)`: Remove the given node or token
+* `removeRange(range)`: Remove text in the given range
+* `replaceText(nodeOrToken, text)`: Replace the text in the given node or token
+* `replaceTextRange(range, text)`: Replace the text in the given range
```

Comment:
Only this description has a full stop(`.`) at the end. We should add it to all for consistency.
```suggestion
* `insertTextAfter(nodeOrToken, text)`: Insert text after the given node or token.
* `insertTextAfterRange(range, text)`: Insert text after the given range.
* `insertTextBefore(nodeOrToken, text)`: Insert text before the given node or token.
* `insertTextBeforeRange(range, text)`: Insert text before the given range.
* `remove(nodeOrToken)`: Remove the given node or token.
* `removeRange(range)`: Remove text in the given range.
* `replaceText(nodeOrToken, text)`: Replace the text in the given node or token.
* `replaceTextRange(range, text)`: Replace the text in the given range.
```

Reply:
Is this the final thing in between this PR and it being merged?

Reply:
It looks like it, I'll just merge these and then merge the PR.